.github/workflows/mac-cpu-arm64-build.yml

```
@@ -36,26 +36,50 @@ jobs:
           mv ${{ env.ORT_PACKAGE_NAME }}/build/native/include ort/
           mv ${{ env.ORT_PACKAGE_NAME }}/runtimes/osx-arm64/native/* ort/lib/
 
-
       - name: Configure CMake
         run: |
-          cmake --preset macos_cpu_release
+          cmake --preset macos_arm64_cpu_release
 
       - name: Build with CMake
         run: |
-          cmake --build --preset macos_cpu_release --parallel
+          cmake --build --preset macos_arm64_cpu_release --parallel
         continue-on-error: false
 
+      - name: Install the python wheel and test dependencies
+        run: |
+          python3 -m venv genai-macos-venv
+          source genai-macos-venv/bin/activate
+          python3 -m pip install -r test/python/requirements.txt
+          python3 -m pip install -r test/python/requirements-macos.txt
+          python3 -m pip install build/cpu/osx-arm64/wheel/onnxruntime_genai*.whl
```

To avoid multiple onnxruntime packages from being downloaded (ort-nightly from the nightly feed and onnxruntime from the stable feed).

```suggestion
          python3 -m pip install build/cpu/osx-arm64/wheel/onnxruntime_genai*.whl --no-deps
```

---

Yeah sure.